src/parser/name_resolution.c

```diff
@@ -3630,6 +3641,13 @@ pt_find_name_in_spec (PARSER_CONTEXT * parser, PT_NODE * spec, PT_NODE * name)
 	      name->info.name.default_value = pt_dbval_to_value (parser, &val);
 	      PT_NAME_INFO_SET_FLAG (name, PT_NAME_DEFAULTF_ACCEPTS);
 	    }
+#if 0
+	  else if (spec->info.spec.derived_table_type == PT_DERIVED_DBLINK_TABLE)
+	    {
+	      // TODO: ctshim_assert
+	      ;
+	    }
+#endif
```

Comment:
This part may require further code review later, so I left a trail.
Please refer to it during the review.